package.json

```diff
@@ -21,5 +21,15 @@
   "bugs": {
     "url": "https://github.com/SoftwareBrothers/admin-bro-expressjs/issues"
   },
-  "homepage": "https://github.com/SoftwareBrothers/admin-bro-expressjs#readme"
+  "homepage": "https://github.com/SoftwareBrothers/admin-bro-expressjs#readme",
+  "dependencies": {
+    "admin-bro": "^0.1.2",
```

Comment:
* admin-bro-mongoose should be in dev dependencies
* admin-bro, express and body-parser should be in peerDependencies

Reply:
Can't run scripts without having admin-bro, express and body-parser in dependencies 😞